server/src/main/java/org/apache/iotdb/db/query/dataset/groupby/GroupByWithValueFilterDataSet.java

```diff
@@ -67,9 +69,9 @@ public GroupByWithValueFilterDataSet(long queryId, List<Path> paths, long unit,
   /**
    * init reader and aggregate function.
    */
-  public void initGroupBy(QueryContext context, List<String> aggres, IExpression expression)
-      throws StorageEngineException, QueryProcessException, IOException {
-    initAggreFuction(aggres);
+  public void initGroupBy(QueryContext context, List<String> aggres, List<TSDataType> dataTypes,
```

Comment:
change this to a private method that called in GroupByWithValueFilterDataSet constructor